Makefile

```diff
@@ -21,17 +21,17 @@ ci:
 	@if [ "$(TEST_SUITE)" = "linter" ]; then make ci-linter; else make ci-tests; fi
 
 ci-tests:
-	go get -u github.com/go-lintpack/lintpack/...
+	go get github.com/go-lintpack/lintpack/...
```

Comment:
I'm not sure you need the `go get` at all here. You're working in module mode. If the dependency is not available it will be fetched.